### PR TITLE
feat: register ajv plugins on init for all ajv instances

### PR DESCRIPTION
**Pull request by ghost**
- Use Ajv's Plugin type for plugins
- Register ajv plugins for all instances on init

### CODE REVIEW

File: packages/http-server/src/AjvValidator.ts [hunk 1]
```diff
@@ -182,6 +183,16 @@ export class AjvValidator<Request = unknown> {
 		}
 	}
 
+	private registerPlugins() {
+		this.options.plugins.forEach(p => {
```
Comment (pierissimo) [inline packages/http-server/src/AjvValidator.ts:187]: `this.options.plugins` may be undefined, better to add a check there

**Review by pierissimo — CHANGES_REQUESTED** [review]
can you please add some unit tests?

File: packages/http-server/src/AjvValidator.ts (hunk 1 above)
Comment (adriasala124) [reply packages/http-server/src/AjvValidator.ts]: Sure, added optional chaining

**Review by joseandrespg — COMMENTED** [review]
Can we add tests for the change?

File: packages/http-server/src/AjvValidator.ts [hunk 2]
```diff
@@ -182,6 +183,16 @@ export class AjvValidator<Request = unknown> {
 		}
 	}
 
+	private registerPlugins() {
+		this.options.plugins?.forEach(p => {
```
Comment (pierissimo) [inline packages/http-server/src/AjvValidator.ts:187]:
```suggestion
		this.options?.plugins?.forEach(p => {
```

**Review by pierissimo — APPROVED** [review]

**Pull request merged**